docker-compose.yml

```diff
@@ -372,6 +372,26 @@ services:
          - opentsdb.pendev
 
 
+  openkilda-gui:
+    container_name: openkilda-gui
+    build:
+      context: services/src/openkilda-gui
+    image: "kilda/openkilda-gui:${full_build_number:-latest}"
```

Comment:
We do not produce docker image `kilda/openkilda-gui`.

You need or "build" or "image".

Reply:
I followed the same steps like we did for others (floodlight, wfm, neo4j)

Reply:
and?

Reply:
What you refer as Build or Image. Please suggest

Reply:
External/foreign services are build into images into Makefile, our services are build by docker-compose.